app/controllers/poems_controller.rb

```diff
@@ -49,4 +50,18 @@ def poem_params
   def is_mine
     @is_mine = Poem.find(params[:id]).user_id == @current_user.id
   end
+
+  def notify_to_slack
+    text = <<-EOC
+------------------------
+新しいポエムが投稿されました
+
+▼URL
+#{url_for @poem}
+▼Author
+#{current_user.name}
+    EOC
+
+    Slack.chat_postMessage text: text, username: "PoemMaster", channel: ENV['SLACK_CHANNEL']
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [92/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)